app/views/layouts/application.html.erb

```diff
@@ -19,15 +19,15 @@
           <span class="icon-bar"></span>
           <span class="icon-bar"></span>
         </button>
-        <%= link_to 'Time Tracker', '/', class: "navbar-brand" %>
+        <%= link_to 'Time Tracker', '/', data: { keybinding: '1' }, class: "navbar-brand" %>
```

Comment:
This link actually corresponds to Time Entries, and I'd rather not have two keys that do the same thing. What about `1` going to Tags, `2` to Tasks and `3` to Time Entries?